classes/person.rb

```diff
@@ -20,6 +21,10 @@ def can_use_services?
     of_age? || @parent_permission == true
   end
 
+  def add_rental(rental)
+    @rentals.append(rental)
+  end
```

Comment:
@Amazinggracee great work adding the `add_rental` method, however, the method is supposed to take `book, date, self `parameters to help achieve the `the 'has-many' association`, this is similar to what you did in the book class `add_rental` method.